### ISSUE TITLE
A better dashboard for prover registration

### ISSUE THREAD

**Issue by d1onys1us** [taikoxyz/taiko-mono#14795]
### Describe the feature request

The current "prover registration page" is a simple markdown page: https://taiko.xyz/docs/reference/prover-market-page.

It has a few problems:
1. In order for someone to add their prover endpoint, they have to go through the trouble of opening a Pull Request, and getting that approved / merged into the website.
2. There is no sense of "quality" on the endpoints. For example, some of the prover service endpoints are likely to eventually be abandoned and go offline. It would be nice to have something to solve this.

## Recommended solution

Create a separate website, and on this website allow provers to register with a form. The form input will take the prover address, and make sure it actually works by doing a liveness check (sending some hashed txList and receives a bond). Only after confirming the prover is online, will the website add the prover.

Some further ideas include some "likes" or thumbs up / thumbs down, but I think the most important part is just doing some liveness checks (especially at an interval) to clean up the page and make sure they are all good to go. Instead of removing endpoints, there could perhaps be an extra column that shows something like "Is healthy" | "No response" (in green / red).

### Describe alternatives you've considered

Description of the alternatives you've considered here.

### Additional context

Additional context here.

**Comment by fahimMuhamma** [comment]
Yes it is

**Comment by SrijanShovit** [comment]
Hey, @d1onys1us @fahimMuhamma I am interested in working on this issue. Could you please assign it to me? Thank you!!

**Comment by wolfderechter** [comment]
Side note: since the prover endpoints are http out of the box you might run into the same issue as me. For the taiko node dashboard (https://github.com/wolfderechter/taiko-node-dashboard) hosted version (http://dojonode.xyz/#/dashboard) I’m forced to host it as http or else I get errors about mixing http and https because of the connection to the http node. I built it with (client side) svelte, so maybe using some server side could fix this but honestly no idea atm

**Comment by d1onys1us** [comment]
> Hey, @d1onys1us @fahimMuhamma I am interested in working on this issue. Could you please assign it to me? Thank you!!

Thanks for your interest @SrijanShovit, I have assigned the issue to you. I would recommend posting here your implementation plan so that we can give you some review, answer questions, or provide any useful feedback. Let us know if we can help in any way :)

**Comment by Ersoy1971** [comment]
I like to b part of it

**Comment by wolfderechter** [comment]
I think it would be beneficial if  everyone can add their own endpoint without logging in. But only the endpoints that pass the healthcheck are shown in the table. This way deprecated endpoints automatically get hidden as soon as possible.

And not sure if possible but would be really useful if we the minimum fee can be displayed dynamically so if the prover changes this, it automatically gets updated in the table. But this might need some changes in the prover

**Comment by spachib** [comment]
is there any way i can have a healthy prover automatically picked for me when i go to propose?

**Comment by entrepreneurlibre** [comment]
A more user-friendly interface will be welcome

**Comment by d1onys1us** [comment]
hey @SrijanShovit any updates here? i'm also going to tag in some other people:
- @RogerLamTd any updates? i think you were also looking at this
- @wolfderechter and gang (idk helios @ ) -> do you think there's anywhere in dojo dashboard where this would be good to add?

**Comment by wolfderechter** [comment]
>    @wolfderechter and gang (idk helios @ ) -> do you think there's anywhere in dojo dashboard where this would be good to add?

Looked into it and dojo node is down to create a better permissionless prover market page! Seems to be doable, especially with the latest `/status` endpoint in the clients! @d1onys1us

**Comment by RogerLamTd** [comment]
> hey @SrijanShovit any updates here? i'm also going to tag in some other people:
> 
> * @RogerLamTd any updates? i think you were also looking at this
> * @wolfderechter and gang (idk helios @ ) -> do you think there's anywhere in dojo dashboard where this would be good to add?

Not much to update from my end, I'm pretty new to svelte so progress is slow; I'll probably work on it on the side as a project so happy to hand it off!

**Comment by samui68400** [comment]
I've reviewed the issue regarding the "prover registration page" and am keen to contribute. I believe I can add value to this task. While I see SrijanShovit is already on it, I'm open to collaborating or assisting in any way possible. Can we discuss how I might be involved?

**Comment by ldcryptoqueen** [comment]
🫣🏆

**Comment by foodara90** [comment]
gas fee

**Comment by bollyshot** [comment]
👀

**Comment by aniketbull** [comment]
when🔥

**Comment by xynity** [comment]
lets see👀🔥🏃

**Comment by paxden007** [comment]
ok

**Comment by ulteus** [comment]
round 1

**Comment by CrCollecter** [comment]
cool

**Comment by foodara90** [comment]
Time to launch Taiko

**Comment by ldcryptoqueen** [comment]
🏆🫣

**Comment by Cryptodelhi** [comment]
Waiting

**Comment by oystel1** [comment]
Best is taiko🙌

**Comment by mixxos** [comment]
👀✨

**Comment by syreoss** [comment]
♂️🛡️
